### ISSUE TITLE
[Markdown toolbar for reddit.com] <Doesn't seem to work on old.reddit>

### ISSUE THREAD

**Issue by Minibus93** [darkred/Userscripts#45]
### Please ensure:

- [X] I included in the issue title the script name I'm referring to.
- [X] I performed a search of the [issue tracker](https://github.com/darkred/Userscripts/issues) and in the relevant Greasyfork [feedback page](https://greasyfork.org/en/users/2160-darkred?sort=updated) to avoid opening a duplicate issue.
- [X] The bug is caused by the script itself. It doesn't happen if I disable the script, or in a fresh browser profile.

### Include in this issue:

- [ ] Screenshots/video/gif demonstrating the bug, if it’s visual
- [ ] Console errors, if any

### Describe the problem and how to replicate it

The script doesn't seem to work on old reddit.

Install "Markdown toolbar for reddit.com" (I'm using violentmonkey)

Try to comment anything while using https://old.reddit.com/

See that there's no markdown bar

### Example URL

https://old.reddit.com/

### Script Manager

Violentmonkey 2.13.0

### Browser(s) used

Firefox 95.0.2

**Comment by darkred** [comment]
Thanks for the report. I have stopped using this script a long time now, as I use the Reddit Enhancement Suite extension.
What you reported can be fixed merely by adding two `@include` lines.
But there's now a problem which I doubt I can fix: the default placeholder becomes momentarily visible whenever one of the buttons is clicked. e.g. screenshot from /r/Crackwatch :
![12345~~4](https://user-images.githubusercontent.com/723651/148415454-06dd50b3-f446-4398-bc0e-2f172e92a602.gif)

I pushed these changes but just de-listed the script on GreasyFork.
